specification/azurefleet/AzureFleet.Management/tspconfig.yaml

```diff
@@ -37,3 +37,13 @@ options:
     service-name: Compute Fleet
     examples-directory: "examples"
     flavor: azure
+  "@azure-tools/typespec-go":
+    service-dir: "sdk/resourcemanager/computefleet"
+    package-dir: "armcomputefleet"
+    module: github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}
+    flavor: "azure"
+    generate-fakes: true
+    head-as-boolean: true
+    inject-spans: true
+    remove-unreferenced-types: true
+    examples-directory: ./examples
```

Comment:
add several configs: 
generate-examples: true
fix-const-stuttering: true

Reply:
@tadelesh , is `generate-examples` a new option of go emitter? I don't find it in the code.

Reply:
Added them as release tool won't do schema validation for emitter options. This option will be added later to emitter code.